addon/mixins/route.js

```diff
@@ -217,6 +217,10 @@ export default Ember.Mixin.create({
     return false;
   },
 
+  updateModel(model, infinityModel) {
```

Comment:
We also don't need to pass `infinityModel` here, we can get it with `this.get('_modelPath')`.

Can we also rename this method to `addObjects`, and the first arg to `objects`?  Want to be explicit that we aren't updating or changing the infinityModel in place, we're just adding to it.

Also - a small note in the README would be great!